src/main/java/io/stargate/sgv2/jsonapi/exception/SchemaException.java

```diff
@@ -48,6 +50,7 @@ public enum Code implements ErrorCode<SchemaException> {
 
     // older below - seperated because they need to be confirmed
     INVALID_CONFIGURATION,
+    INVALID_FORMAT_FOR_INDEX_CREATION_COLUMN,
```

Comment:
nit: move this to be above - this is the list of things to remove at some point :)